backend/build.gradle

```diff
@@ -0,0 +1,50 @@
+apply plugin: 'fatjar'
+
+version = "0.0.1"
+
+allprojects {
+  apply plugin: 'java'
+  sourceCompatibility = 1.6
+  targetCompatibility = 1.6
+}
+
+buildscript {
+  dependencies {
+    classpath 'eu.appsatori:gradle-fatjar-plugin:0.1.2' // adds fatJar task
+  }
+  repositories {
+    mavenCentral()
+  }
+}
+
+dependencies {
+  compile 'com.sendgrid:sendgrid-java:1.1.0'
+  implementation group: "com.twilio.sdk", name: "twilio", version: "7.52.0"
+  //compile files('/Users/scottmotte/code/active/sendgrid/sendgrid-java/build/libs/sendgrid-1.1.0-jar.jar')
+}
+
+repositories {
+  mavenCentral()
+}
+
+// adds 'with-dependencies' to the fatJar name
+fatJar {
+  classifier 'jar'
+  baseName "scottmotte"
+  manifest {
+    attributes("Implementation-Title": "SendgridJavaExample", "Implementation-Version": version)
+  }
+}
+
+// copy fatJar to base project directory so they will be in git (and on github for download)
+build << {
+  copy {
+    println "Copying ${fatJar.archiveName} to $projectDir/repo/com/github/scottmotte/$version"
+    from("$buildDir/libs/${fatJar.archiveName}")
+    into("$projectDir/repo/com/github/scottmotte/$version")
+  }
+}
+
+artifacts {
+  archives fatJar
+}
```

Comment:
I'm not exactly sure how Gradle works, but I think it's an alternative to Maven (what we're using now). Is there any way we can switch this to Maven in our pom.xml file instead of Gradle in this build.gradle file?

Reply:
+1

Reply:
I'm also not too familiar with cradle, I have a form of Twilio in my pom.xml so I think that should be enough